net/deluge/Portfile

```diff
@@ -138,8 +114,30 @@ variant python310 conflicts python27 python39 description {Build for Python 3.10
                     port:py310-zopeinterface
 }
 
-if {![variant_isset python27] && ![variant_isset python38] && ![variant_isset python39] && ![variant_isset python310]} {
-    default_variants    +python310
+variant python311 conflicts python38 python39 python310 description {Build for Python 3.11} {
+        require_active_variants libtorrent-rasterbar python311
+        python.default_version 311
+        depends_build-append \
+                    port:py310-setuptools \
+                    port:py310-wheel
+        depends_lib-append \
+                    port:py310-gobject3 \
+                    port:py310-service_identity \
+                    port:py310-twisted \
+                    port:py310-asn1 \
+                    port:py310-rencode \
+                    port:py310-xdg \
+                    port:py310-openssl \
+                    port:py310-Pillow \
+                    port:py310-mako \
+                    port:py310-chardet \
+                    port:py310-six \
+                    port:py310-setproctitle \
+                    port:py310-zopeinterface
```

Comment:
It looks like you copy-pasted the dependencies from the python310 variant, without updating.

Speaking of copy-pasta, all of the python-related logic could be simplified, by encapsulating into a proc. This is what I do for `glade`:

```
proc py_setup {python_branch} {
    global prefix frameworks_dir
    set python_version          [string map {"." ""} ${python_branch}]
    set python_framework        ${frameworks_dir}/Python.framework/Versions/${python_branch}

    configure.args-replace      -Dpython=disabled -Dpython=enabled
    configure.python            ${prefix}/bin/python${python_branch}
    configure.env-append        "PYTHON_LIBS=-L${python_framework}/lib -lpython${python_branch}" \
                                PYTHON_LIB_LOC=${python_framework}/lib
    depends_lib-append          port:py${python_version}-gobject3
    configure.pkg_config_path   ${python_framework}/lib/pkgconfig
}

variant python38 conflicts python39 python310 description {Build Python 3.8 widgets support} {
    py_setup 3.8
}

variant python39 conflicts python38 python310 description {Build Python 3.9 widgets support} {
    py_setup 3.9
}

variant python310 conflicts python38 python39 description {Build Python 3.10 widgets support} {
    py_setup 3.10
}
```

Reply:
I have no eyes -_-

Gonna fix this now, using your suggestion.

Reply:
Okay, another go. See if this works out.